validation-test/SIL/crashers/cse-open-existential-ref-term-inst.sil

```diff
@@ -0,0 +1,27 @@
+// RUN: not --crash %target-sil-opt -enable-sil-verify-all %s -cse
```

Comment:
This is another case that gets misoptimized (see the `FIXME`). I just added a crasher for now because I want to focus on landing a fix for the GitHub issue. We don’t need a `REQUIRES: asserts` here, do we?

Reply:
I don't think -sil-verify-all need asserts.